server/api/auth/login/ldap.post.ts

```diff
@@ -32,21 +34,22 @@ function getSSLConfig(): false | { rejectUnauthorized: boolean; ca: string | Buf
   }
 
   return {
-    rejectUnauthorized: config.ldap_ssl_insecure !== "true",
+    rejectUnauthorized: !config.ldap_ssl_insecure,
     ca: sslCA,
   };
 }
 
 export default defineEventHandler(async (event) => {
-    console.log("entering ldap flow");
+  console.log("entering ldap flow");
```

Comment:
i left all the console logs in there cause i was too lazy to remove them, but we probably should.  or at least replace them with whatever logging library is being used.

Reply:
I continued the lazy.  I was worried about adding too much with the auth switch so close on the horizon, but you're right, this using the logger is very low effort.